config/jobs/build-drivers/build-drivers.yaml

```diff
@@ -36,6 +36,78 @@ presubmits:
         imagePullPolicy: Always
         securityContext:
           privileged: true
+  - name: build-drivers-centos-presubmit
+    decorate: true
+    skip_report: false
+    agent: kubernetes
```

Comment:
let's add a run if changed section here on all of these presubmits, so it doesnt run a bunch of 3Hour jobs on every PR. Here's a few examples of how to format that. 

for just driverkit config
`run_if_changed: '^driverkit/config/'`

or for config and utils
`run_if_changed: ^(driverkit/config/|driverkit/utils/)`

Reply:
great idea! let's do it

Reply:
maybe `driverkit/config/` is enough, WDYT @leodido ?

Reply:
I'm cool with whatever.

Reply:
My regex isn't the best.... lol will this will run on anything in `driverkit/` except the README?

Reply:
yess :)

Reply:
Maybe regex negative look-ahead is not supported by Golang, I think we can let `checkconfig` find out :)